<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IR Lib Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="irTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="irEncodingTab">
     <attribute name="title">
      <string>Encoding</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="irDecodingTab">
     <attribute name="title">
      <string>Decoding</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>IR Code</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="decChigo1RadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>1000 (Chigo1)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="decAux1RadioButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>1001 (Aux1)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="decTcl1RadioButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>1002 (TCL1)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="decChigo2RadioButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>1003 (CHIGO3)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decDecodePushButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Decode</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="decIRPacketTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>1171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTextEdit" name="decIRJsonTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>1171</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTextEdit" name="decIRCodeTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1171</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="irFindRemoteTab">
     <attribute name="title">
      <string>Find Remote</string>
     </attribute>
     <widget class="QLineEdit" name="frBarcodeLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter AC Barcode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="frFindPushButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Found Remote ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="frFoundIdLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
